main.go

```diff
@@ -57,9 +56,13 @@ func backendMain() error {
 		return errors.Errorf("unable to create engine client: %v", err)
 	}
 
-	engine, err := core.GetEngine()
-	if err != nil {
-		return errors.Errorf("unable to get engine: %s", err)
+	var engine *core.Engine
+	if !loadedConfig.EngineDisabled {
```

Comment:
Are we using engine only for notifications? Because if not, we might get panic because of this.